### PR TITLE
Pal 252 option include k8s dashboard

### PR DESCRIPTION
**Pull request by dev-958**

### CODE REVIEW

**Review by nw1984 — DISMISSED** [review]

File: charts/dashboard/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Crown Copyright
```
Comment (ac74475) [inline charts/dashboard/Chart.yaml]:
```suggestion
# Copyright 2020 Crown Copyright
```

File: charts/dashboard/templates/ingress-route.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: traefik.containo.us/v1alpha1
```
Comment (ac74475) [inline charts/dashboard/templates/ingress-route.yaml:15]: Add copyright and licence heeader

File: charts/dashboard/templates/pathparam-regex.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+apiVersion: traefik.containo.us/v1alpha1
```
Comment (ac74475) [inline charts/dashboard/templates/pathparam-regex.yaml:15]: add copyright and licence header

File: charts/dashboard/values.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Crown Copyright
```
Comment (ac74475) [inline charts/dashboard/values.yaml]:
```suggestion
# Copyright 2020 Crown Copyright
```

**Review by ac74475 — CHANGES_REQUESTED** [review]

**Review by ac74475 — APPROVED** [review]

**Pull request merged**